modules/apps/notification/notification-service/src/main/java/com/liferay/notification/internal/type/EmailNotificationType.java

```diff
@@ -252,22 +253,35 @@ public void sendNotification(NotificationContext notificationContext)
 
 		siteDefaultLocale = portal.getSiteDefaultLocale(groupId);
 
-		userLocale = user.getLocale();
-
 		NotificationTemplate notificationTemplate =
 			notificationContext.getNotificationTemplate();
 
-		String body = _formatBody(
-			notificationTemplate.getBodyMap(), userGroup, notificationContext);
 		NotificationRecipient notificationRecipient =
 			notificationTemplate.getNotificationRecipient();
-		String subject = formatLocalizedContent(
-			notificationTemplate.getSubjectMap(), notificationContext);
 
 		Map<String, Object> notificationRecipientSettings =
 			NotificationRecipientSettingUtil.toMap(
 				notificationRecipient.getNotificationRecipientSettings());
 
+		String useMostRelevantLocaleForGuestUsers = _getBooleanValue(
+			NotificationRecipientSettingConstants.
+				NAME_USE_MOST_RELEVANT_LOCALE_FOR_GUEST_USERS,
```

Comment:
Hey @pedro-oliveira446 just rename it to `NAME_USE_PREFERRED_LOCALE_FOR_GUEST_USERS`